Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,34 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  font-size: $text-small;
+  height: 320px;
+  padding: 20px 30px;
+  text-align: center;
+  width: 220px;
+
+  img {
+    height: $tech-logo-size;
+  }
+
+  h2 {
+    font-size: $text-medium;
+    margin-top: 20px;
```

Comment:
try whenever possible to use margin bottom instead of top because that way you'll see the element with the margin first and it's easier to understand